cnxdb/archive-sql/schema/triggers.sql

```diff
@@ -90,13 +90,13 @@ CREATE TRIGGER delete_from_latest_version
   EXECUTE PROCEDURE delete_from_latest();
 
 
-
-
 CREATE OR REPLACE FUNCTION post_publication() RETURNS trigger AS $$
 BEGIN
-  PERFORM pg_notify('post_publication', '{"module_ident": '||NEW.module_ident||', "ident_hash": "'||ident_hash(NEW.uuid, NEW.major_version, NEW.minor_version)||'", "timestamp": "'||CURRENT_TIMESTAMP||'"}');
-  RETURN NEW;
-END;
+      IF TG_OP = 'INSERT' OR ( TG_OP = 'UPDATE' AND OLD.stateid != 5 ) THEN
```

Comment:
Can you put a comment here to document what this means in general terms? I'm guessing that somebody will come across this in the future then need to do translation to determine what is actually happening. It's not bad in the context of the pull request, but probably a little confusing without the context.

Reply:
Yeah, I've gone back and forth on using the integers and joining against the 'state' table to use the actual names of the states (slightly more self documenting ) I'll add comments as you suggest.